application/values.yaml

```diff
@@ -429,8 +429,8 @@ rbac:
 
   # Service Account to use by pods
   serviceAccount:
-    enabled: false
-    name: ""
+    enabled: true
```

Comment:
The purpose of having SA enabled by default is again a best practice by default thing, best practice is that every deployment should bring its own SA and  not mount the default SA's